src/engine/DebugOverlays.Inspector.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Godot;
+
+public partial class DebugOverlays
```

Comment:
This partial class needs a summary comment here. The StyleCop warning should be on but I don't know why it doesn't trigger here.